homeassistant/util/color.py

```diff
@@ -1,5 +1,6 @@
 """
-homeassistant.util.color
+homeassistant.util.color.
```

Comment:
If you're going to follow pep257, it should be a description instead of package name.